.github/workflows/sims-cron.yml

```diff
@@ -0,0 +1,70 @@
+name: Sims Cron Job
```

Comment:
Maybe we could name this `Sims Nightly` and the file `sims-nightly.yml` to be more specific?